toucan_connectors/hubspot/helpers.py

```diff
@@ -0,0 +1,104 @@
+from enum import Enum
+from typing import List
+
+import pandas as pd
+
+
+# HubspotDataset is defined in the helpers to avoid a cyclic import
+class HubspotDataset(str, Enum):
+    contacts = 'contacts'
+    companies = 'companies'
+    deals = 'deals'
+    products = 'products'
+    webanalytics = 'web-analytics'
+    emails_events = 'emails-events'
+
+
+def hubspot_api_format_v3(data: List[dict]) -> pd.DataFrame:
```

Comment:
```suggestion
def format_hubspot_api_v3(data: List[dict]) -> pd.DataFrame:
```
 ? pour reprendre le meme genre de nommage que les autres fonctions de formattage

Reply:
Fixé  : https://github.com/ToucanToco/toucan-connectors/pull/340/commits/f75b0319ab910501e34378b4a23036997c6a3853#diff-b0bbc03df068e102cbea31a2104a9b3044f4911ca41eb9fbd75a0ddd675b58faL8